client.go

```diff
@@ -1738,26 +1741,28 @@ func (c *ToznySDKV3) GetOrCreateNamespace(ctx context.Context, options Namespace
 			return nil, err
 		}
 		group = createGroupResponse
-		// get membership keys
-		membershipKeyRequest := storageClient.CreateMembershipKeyRequest{
-			GroupAdminID:      c.StorageClient.ClientID,
-			NewMemberID:       c.StorageClient.ClientID,
-			EncryptedGroupKey: createGroupResponse.EncryptedGroupKey,
-			ShareePublicKey:   c.StorageClient.EncryptionKeys.Public.Material,
-		}
-		membershipKeyResp, err := c.StorageClient.CreateGroupMembershipKey(ctx, membershipKeyRequest)
-		if err != nil {
-			return nil, err
-		}
-		// make the capabilities
+		// make the capabilities -- for now, all members get read and share capabilities
 		groupMemberCapabilities := []string{storageClient.ShareContentGroupCapability, storageClient.ReadContentGroupCapability}
-		// make a new member
+		// make new members from the clientIDs
 		memberRequest := []storageClient.GroupMember{}
-		memberRequest = append(memberRequest, storageClient.GroupMember{
```

Comment:
Do we need to make sure that calling client is made a member of the group? Or if the callee doesn't include themselves in the options.ClientIDs will they not be a member?